<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportClientNomenclaturePlanSetupDialog</class>
 <widget class="QDialog" name="ReportClientNomenclaturePlanSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="5" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbSocStatusClass"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblSocStatusClass">
     <property name="text">
      <string>Класс соц. статуса</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="pnlDate" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="lblBegDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Дата &amp;начала периода</string>
        </property>
        <property name="buddy">
         <cstring>edtBegDate</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Дата:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CDateEdit" name="edtBegDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QComboBox" name="cmbDateType">
        <item>
         <property name="text">
          <string>Выполнения</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Назначения</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Начала</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Планирования</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="CDateEdit" name="edtEndDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblEndDate">
        <property name="text">
         <string>Дата &amp;окончания периода</string>
        </property>
        <property name="buddy">
         <cstring>edtEndDate</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbEventPurpose"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbEventResult"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblEventResult">
     <property name="text">
      <string>Результат</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbSocStatusType"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblSetPerson">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Назначил</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblEventTypePurpose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Назначение</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblSocStatusType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Тип соц. статуса</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="COrgStructureComboBox" name="cmbSetOrgStructure"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Выполнил</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>ТМЦ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lblNomenclatureFeatureName">
        <property name="text">
         <string>Характеристика</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbNomenclatureFeatureValue"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbNomenclatureFeatureName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblNomenclatureFeatureValue">
        <property name="text">
         <string>Значение</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblNomenclatureType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Тип</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CRBComboBox" name="cmbNomenclatureType"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbDateType</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbSetOrgStructure</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>cmbSocStatusClass</tabstop>
  <tabstop>cmbSocStatusType</tabstop>
  <tabstop>cmbEventPurpose</tabstop>
  <tabstop>cmbEventResult</tabstop>
  <tabstop>cmbNomenclatureType</tabstop>
  <tabstop>cmbNomenclatureFeatureName</tabstop>
  <tabstop>cmbNomenclatureFeatureValue</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportClientNomenclaturePlanSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportClientNomenclaturePlanSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
